resources/js/core.js

```diff
@@ -15,7 +15,8 @@ const photoBooth = (function () {
                 height: 480,
                 facingMode: 'user',
             }
-        };
+        },
+        collageLimit = 4;
```

Comment:
Now we have to limits. One in the php script and one here. Such thing should be maintained at one place.

Reply:
Done - can you please review the changes again?